### ISSUE TITLE
EACCESS: permission denied :(

### ISSUE THREAD

**Issue by coop** [elixir-tools/next-ls#382]
Twice I have had the issue where I start neovim and see the error:

```
Spawing language server with cmd: `/path/to/nextls` failed with error message: EACCESS: permission denied
```

This happened after restarting vim because nextls decided to ignore my formatting instructions and added parens to all field definitions in schema files.

To "fix" I `rm /path/to/nextls`, open vim, press "1" to install nextls and we're good. I have this seen on a 0.17.x release. I am running 0.18.x now.

I am running lunarvim 1.3 and here is my full configuration:

```
lvim.plugins = {
  {
    "vim-test/vim-test",
    config = function()
      vim.g["test#strategy"] = "kitty"
    end,
  },
  { "duggiefresh/vim-easydir" },
  { "tpope/vim-eunuch" },
  {
    "elixir-tools/elixir-tools.nvim",
    version = "*",
    event = { "BufReadPre", "BufNewFile" },
    config = function()
      local elixir = require("elixir")

      elixir.setup {
        nextls = {
          enable = true,
          spitfire = true,
          version = "0.18.0",
          init_options = {
            experimental = {
              completions = {
                enable = true, -- control if completions are enabled. defaults to false
              },
            },
          },
        },
        elixirls = {
          enable = false,
          settings = {
            enableTestLenses = true,
          },
        },
        credo = {
          enable = false,
        },
      }
    end,
    dependencies = {
      "nvim-lua/plenary.nvim",
    },
  },
}
```

**Comment by mhanberg** [comment]
I'll have to investigate. Same thing happened to me but I assumed it was due to my own error.

**Comment by mhanberg** [comment]
Fixed by 69db3b2b82881a7e0dca502420f8d16334d3933a
